tests/kernel/install_ltp.pm

```diff
@@ -192,11 +193,11 @@ curl --form upload=\@/root/openposix-test-list-$tag --form target=assets_public
 
 sub install_from_git {
     my ($tag) = @_;
-    my $url = get_var('LTP_GIT_URL') || 'https://github.com/linux-test-project/ltp';
-    my $rel = get_var('LTP_RELEASE') || '';
-    my $timeout     = check_var('ARCH', 's390x') || check_var('ARCH', 'aarch64') ? 7200 : 1440;
+    my $url         = get_var('LTP_GIT_URL', 'https://github.com/linux-test-project/ltp');
+    my $rel         = get_var('LTP_RELEASE');
+    my $timeout     = (is_aarch64 || is_s390x) ? 7200 : 1440;
     my $configure   = './configure --with-open-posix-testsuite --with-realtime-testsuite';
-    my $extra_flags = get_var('LTP_EXTRA_CONF_FLAGS') || '';
+    my $extra_flags = get_var('LTP_EXTRA_CONF_FLAGS', '');
```

Comment:
You don't need to set default value to `''` and `my $extra_flags = get_var('LTP_EXTRA_CONF_FLAGS');` will be enough. If there is no `LTP_EXTRA_CONF_FLAGS` then `$extra_flags` will be undef.

Reply:
@czerw Yes, `undef` is a reason, why there is a default empty value (originally from Richie). I guess we'd get `Use of uninitialized value in concatenation (.) or string at ...` error. Or am I missing something?

Reply:
ok